pages/account/addressBook.tsx

```diff
@@ -1,9 +1,182 @@
 import AccountBaseTemplate from "@/components/AccountBaseTemplate";
+import { Button } from "@/components/Button";
+import AddressDisplay from "@/components/checkout/AddressDisplay";
+import Spinner from "@/components/Spinner";
+import {
+  UserAddressesDocument,
+  AddressTypeEnum,
+  AddressDetailsFragment,
+  useSetAddressDefaultMutation,
+  useDeleteAddressMutation,
+  useUserAddressesQuery,
+} from "@/saleor/api";
+import { useAuthState } from "@saleor/sdk";
+import { Key } from "react";
 
 const AddressBookPage: React.VFC = () => {
+  const { authenticated } = useAuthState();
+  const { loading, error, data } = useUserAddressesQuery({
```

Comment:
```suggestion
  const { loading, error, data, refetch } = useUserAddressesQuery({
```

Have you tried to use `refetch` from this query hook?